spec/resources/api/v2/user_resource_spec.rb

```diff
@@ -4,17 +4,24 @@
 require './app/resources/api/v2/user_resource'
 
 RSpec.describe Api::V2::UserResource, type: :resource do
-  subject { described_class.new(resource_model, {}) }
+  subject(:resource) { described_class.new(resource_model, {}) }
```

Comment:
how does this work? you define the `subject(:resource)` but then it's not referred to in the `it `sections below. Does `is_expected` magically use the subject?

Reply:
Yeah it's some sort of rspec magic.  `is_expected` seems to know it needs to use the thing defined as the `subject`.  I didn't know this pattern, but I saw it already existed in our tests.  The only change here was to name the subject, which Rubocop insists on when you remove the exception to the rule like I did here in the rubocop config file.